test/mocks/server/transport_socket_factory_context.h

```diff
@@ -5,7 +5,7 @@
 #include "common/secret/secret_manager_impl.h"
 
 #include "test/mocks/api/mocks.h"
-#include "test/mocks/upstream/mocks.h"
+#include "test/mocks/upstream/cluster_manager.h"
```

Comment:
same here. is there a cluster_manager_mocks, or do you need to include all three build targets for this header?